README.md

```diff
@@ -33,6 +33,7 @@ Updating to a newer version of the SDK? See our [release notes](https://github.c
 ### Android
 
 - Android 5.0 (API level 21) and above
+  * Note that attempting to override minSdkVersion to decrease the minimum supported API level will not work due to internal runtime API level validation. Furthermore, Stripe is updating the Terminal Android SDK to support Google’s recently released Android 14 (SDK 34). Please continue to target SDK 33 in the meantime as there are known issues with mPOS devices and TTP when targeting SDK 34. Please track the following [ticket](https://github.com/stripe/stripe-terminal-android/issues/387) for updates on progress.
```

Comment:
nit: Should the "Stripe is updating the Terminal Android..." sentence and everything after it be a note after the `targetSdkVersion = 31` line?